WordPress/src/main/java/org/wordpress/android/ui/notifications/adapters/NotesAdapter.kt

```diff
@@ -0,0 +1,353 @@
+@file:Suppress("DEPRECATION")
+
+package org.wordpress.android.ui.notifications.adapters
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.AsyncTask
+import android.text.Spanned
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.ViewGroup.MarginLayoutParams
+import android.view.ViewTreeObserver
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.text.BidiFormatter
+import androidx.core.view.ViewCompat
+import androidx.recyclerview.widget.RecyclerView
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.datasets.NotificationsTable
+import org.wordpress.android.models.Note
+import org.wordpress.android.models.Note.NoteTimeGroup
+import org.wordpress.android.models.Note.TimeStampComparator
+import org.wordpress.android.ui.comments.CommentUtils
+import org.wordpress.android.ui.notifications.NotificationsListFragmentPage.OnNoteClickListener
+import org.wordpress.android.ui.notifications.adapters.NotesAdapter.NoteViewHolder
+import org.wordpress.android.ui.notifications.blocks.NoteBlockClickableSpan
+import org.wordpress.android.ui.notifications.utils.NotificationsUtilsWrapper
+import org.wordpress.android.util.GravatarUtils
+import org.wordpress.android.util.RtlUtils
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType
+import javax.inject.Inject
+
+class NotesAdapter(
+    context: Context, dataLoadedListener: DataLoadedListener,
+    onLoadMoreListener: OnLoadMoreListener?
+) : RecyclerView.Adapter<NoteViewHolder>() {
+    private val mAvatarSz: Int
```

Comment:
Ref: https://developer.android.com/kotlin/style-guide#naming_2

> Special prefixes or suffixes, like those seen in the examples `name_`, `mName`, `s_name`, and `kName`, are not used except in the case of backing properties.

It'd be better if we rename these variables to Kotlin style.

Reply:
Absolutely! I'll do that even before merging, since it's a pretty trivial change. Thanks for pointing it out 👍

Reply:
Resolved by https://github.com/wordpress-mobile/WordPress-Android/pull/20129/commits/3c02ee26c7a37bce587b7e04f0a0a38022b5208b